pkg/util/github/github.go

```diff
@@ -235,6 +248,28 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 	return nil
 }
 
+func listDirExcluding(dir string, excludeRegexp string) ([]string, error) {
```

Comment:
@budhrg I also don't understand what do you mean format here. function definition seems good to me.